### ISSUE TITLE
Type "String" does not have a function named "toInt"

### ISSUE THREAD

**Issue by Jobians** [tact-lang/tact#1071]
I don't know what problem 
https://docs.tact-lang.org/cookbook/type-conversion/#int-string

Build script running, compiling SimpleCounter   ⏳ Compiling...Tact compilation failed          Error: contracts/simple_counter.tact:65:10: Type "String" does not have a function named "toInt"Line 65, col 10:                                  64 |     let string: String = "26052021";     > 65 |     dump(string.toInt());                                ^~~~~~~~~~~~~~                    66 |

**Comment by novusnota** [comment]
Weird, because everything works there with Tact 1.5.2 (in our CI) and even with Tact 1.5.0 (in Web IDE).

For example,
1. I've clicked on "Open in Web IDE"
2. Added `import "@stdlib/deploy";` on top
3. Added a simple contract with a receiver that only calls the `runMe()` function
4. Everything works

Code:
![image](https://github.com/user-attachments/assets/4cb6d4ae-022a-4ced-b4b8-5e4408e5e7da)

Output:
![image](https://github.com/user-attachments/assets/77d2d845-34b1-4424-af23-2f806c11b61f)
